date_range/views/date_range_view.xml

```diff
@@ -11,7 +11,7 @@
                 <field name="date_end"/>
                 <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
                 <field name="parent_type_id" invisible="1"/>
-                <field name="parent_id"/>
+                <field name="parent_id" on_change="onchange_type_id()"/>
```

Comment:
@ntsirintanis What's the purpose of these "onchange" attributes????

Reply:
@lmignon it serves no purpose, it's a mistake, will be removed right away. Thanks!